### PR TITLE
A new resource: Discord Timestamps

### PR DESCRIPTION
**Pull request by DevMarlin**
My very first resource, if found any grammar issues, I'll fix them. A resource about Discord timestamps, will help people to understand better discord's timestamps and how to use them in BDFD.

### CODE REVIEW

**Review by priyanujgogoi-28 — CHANGES_REQUESTED** [review]
Hello FoDevOfficial,
Looks like you have forgotten to update the `SUMMARY.md` file. Kindly do update the file too!

File: src/resources/discordTimestamps.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
+# Discord Timestamps
+Discord timestamps are used to provide time in multiple formants. The information is given according to the user's timezone and locale. Discord timestamps are built with the Unix Time system, meaning that they are dynamic. These can be used by anyone; This includes users, webhooks, and applications.
```
Comment (priyanujgogoi-28) [inline src/resources/discordTimestamps.md]:
```suggestion
Discord timestamps are used to provide time in multiple formats. The information is given according to the user's timezone and locale. Discord timestamps are built with the Unix Time system, meaning that they are dynamic. These can be used by anyone; This includes users, webhooks, and applications.
```

**Comment by DevMarlin** [comment]
Ready!

File: src/SUMMARY.md [hunk 2]
```diff
@@ -42,6 +42,7 @@
     - [Permissions](./resources/permissions.md)
     - [Security](./resources/security.md)
     - [Time Format](./resources/timeFormat.md)
+    - [Discord Timestamps](./resources/discordTimestamps.md)
```
Comment (priyanujgogoi-28) [inline src/SUMMARY.md]: Please do sort it alphabetically!

File: src/resources/discordTimestamps.md [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+| d | <t:1667219160:d> | 10/31/2022 | Short Date |
+| D | <t:1667219160:D> | October 31, 2022 | Long Date |
+| f | <t:1667219160:f> | October 31, 2022 12:26 AM  | Short Date/Time |
+| F | <t:1667219160:F> | Monday, October 31, 2022 12:26 AM | Long Date/Time |
+| R | <t:1667219160:R> | 27 minutes ago | Relative Time |
+
+> The default style is `f`, if no style provided.
+
+## BDFD usability
+
```
Comment (priyanujgogoi-28) [inline src/resources/discordTimestamps.md:21]:
```suggestion

```

File: src/resources/discordTimestamps.md [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+The only functions that can return Unix Time are: [`$getTimestamp`](https://nilpointer-software.github.io/bdfd-wiki/bdscript/getTimestamp), [`$hostingExpireTime`](https://nilpointer-software.github.io/bdfd-wiki/bdscript/hostingExpireTime), and [`$premiumExpireTime`](https://nilpointer-software.github.io/bdfd-wiki/bdscript/premiumExpireTime).
+
+### Example
+
+#### Code:
+
```
Comment (priyanujgogoi-28) [inline src/resources/discordTimestamps.md]:
```suggestion

```

File: src/resources/discordTimestamps.md [hunk 5]
```diff
@@ -0,0 +1,33 @@
...
+```
+$nomention
+<t:$getTimestamp:D>
+```
+#### Output:
+
```
Comment (priyanujgogoi-28) [inline src/resources/discordTimestamps.md]:
```suggestion

```

File: src/resources/discordTimestamps.md [hunk 6]
```diff
@@ -0,0 +1,33 @@
...
+| D | <t:1667219160:D> | October 31, 2022 | Long Date |
+| f | <t:1667219160:f> | October 31, 2022 12:26 AM  | Short Date/Time |
+| F | <t:1667219160:F> | Monday, October 31, 2022 12:26 AM | Long Date/Time |
+| R | <t:1667219160:R> | 27 minutes ago | Relative Time |
+
+> The default style is `f`, if no style provided.
+
+## BDFD usability
+
+The only functions that can return Unix Time are: [`$getTimestamp`](https://nilpointer-software.github.io/bdfd-wiki/bdscript/getTimestamp), [`$hostingExpireTime`](https://nilpointer-software.github.io/bdfd-wiki/bdscript/hostingExpireTime), and [`$premiumExpireTime`](https://nilpointer-software.github.io/bdfd-wiki/bdscript/premiumExpireTime).
```
Comment (priyanujgogoi-28) [inline src/resources/discordTimestamps.md]:
```suggestion
Functions which returns UNIX timestamp:
- [`$getTimestamp`](../bdscript/getTimestamp.md)
- [`$hostingExpireTime`](../bdscript/hostingExpireTime.md)
- [`$premiumExpireTime`](../bdscript/premiumExpireTime.md)
```

**Review by priyanujgogoi-28 — CHANGES_REQUESTED** [review]

File: src/resources/discordTimestamps.md (hunk 1 above)
Comment (priyanujgogoi-28) [reply src/resources/discordTimestamps.md]: Also, in my opinion, the description could have more better explanation than the current one tho.

**Comment by DevMarlin** [comment]
Commited!

File: src/resources/discordTimestamps.md [hunk 7]
```diff
@@ -0,0 +1,33 @@
+# Discord Timestamps
+Discord timestamps are used to provide time in multiple formats. The information is given according to the user's timezone and locale. Discord timestamps are built with the Unix Time system, meaning that they are dynamic. These can be used by anyone; This includes users, webhooks, and applications.
+## Styling
```
Comment (priyanujgogoi-28) [inline src/resources/discordTimestamps.md]:
```suggestion

## Styles
```

File: src/resources/discordTimestamps.md [hunk 8]
```diff
@@ -0,0 +1,33 @@
+# Discord Timestamps
+Discord timestamps are used to provide time in multiple formats. The information is given according to the user's timezone and locale. Discord timestamps are built with the Unix Time system, meaning that they are dynamic. These can be used by anyone; This includes users, webhooks, and applications.
+## Styling
+
```
Comment (priyanujgogoi-28) [inline src/resources/discordTimestamps.md:3]:
```suggestion
```

File: src/resources/discordTimestamps.md [hunk 9]
```diff
@@ -0,0 +1,33 @@
+# Discord Timestamps
+Discord timestamps are used to provide time in multiple formats. The information is given according to the user's timezone and locale. Discord timestamps are built with the Unix Time system, meaning that they are dynamic. These can be used by anyone; This includes users, webhooks, and applications.
+## Styling
+
+Timestamp syntax: `<t:unixTime:Style>`
```
Comment (priyanujgogoi-28) [inline src/resources/discordTimestamps.md:5]:
```suggestion
```

File: src/resources/discordTimestamps.md [hunk 10]
```diff
@@ -0,0 +1,33 @@
+# Discord Timestamps
+Discord timestamps are used to provide time in multiple formats. The information is given according to the user's timezone and locale. Discord timestamps are built with the Unix Time system, meaning that they are dynamic. These can be used by anyone; This includes users, webhooks, and applications.
+## Styling
+
+Timestamp syntax: `<t:unixTime:Style>`
+
```
Comment (priyanujgogoi-28) [inline src/resources/discordTimestamps.md:6]:
```suggestion
Here's a list of all supported time format styles.

```

File: src/resources/discordTimestamps.md [hunk 11]
```diff
@@ -0,0 +1,33 @@
...
+| :---: | :---: | :---: | :---: |
+| t | <t:1667219160:t> | 12:26 AM  | Short Time |
+| T | <t:1667219160:T> | 12:26:00 AM | Long Time |
+| d | <t:1667219160:d> | 10/31/2022 | Short Date |
+| D | <t:1667219160:D> | October 31, 2022 | Long Date |
+| f | <t:1667219160:f> | October 31, 2022 12:26 AM  | Short Date/Time |
+| F | <t:1667219160:F> | Monday, October 31, 2022 12:26 AM | Long Date/Time |
+| R | <t:1667219160:R> | 27 minutes ago | Relative Time |
+
+> The default style is `f`, if no style provided.
```
Comment (priyanujgogoi-28) [inline src/resources/discordTimestamps.md]:
```suggestion
> 📌 The default style is `f`, if no style provided.

```

File: src/resources/discordTimestamps.md [hunk 12]
```diff
@@ -0,0 +1,33 @@
...
+
+## BDFD usability
```
Comment (priyanujgogoi-28) [inline src/resources/discordTimestamps.md]:
```suggestion
## Usability

```

File: src/resources/discordTimestamps.md [hunk 13]
```diff
@@ -0,0 +1,33 @@
...
+Functions which return UNIX timestamp:
+- [`$getTimestamp`](../bdscript/getTimestamp.md)
+- [`$hostingExpireTime`](../bdscript/hostingExpireTime.md)
+- [`$premiumExpireTime`](../bdscript/premiumExpireTime.md)
+
+### Example
+
```
Comment (priyanujgogoi-28) [inline src/resources/discordTimestamps.md]:
```suggestion
```

File: src/resources/discordTimestamps.md [hunk 14]
```diff
@@ -0,0 +1,33 @@
...
+#### Code:
```
Comment (priyanujgogoi-28) [inline src/resources/discordTimestamps.md]:
```suggestion
```

File: src/resources/discordTimestamps.md [hunk 15]
```diff
@@ -0,0 +1,33 @@
...
+```
+$nomention
+<t:$getTimestamp:D>
+```
+#### Output:
```
Comment (priyanujgogoi-28) [inline src/resources/discordTimestamps.md]:
```suggestion
```

**Review by priyanujgogoi-28 — CHANGES_REQUESTED** [review]

File: src/resources/discordTimestamps.md (hunk 9 above)
Comment (priyanujgogoi-28) [reply src/resources/discordTimestamps.md:5]: Add the syntax above styles heading.

**Comment by DevMarlin** [comment]
Commited!

File: src/resources/discordTimestamps.md [hunk 16]
```diff
@@ -0,0 +1,35 @@
...
+| T | <t:1667219160:T> | 12:26:00 AM | Long Time |
+| d | <t:1667219160:d> | 10/31/2022 | Short Date |
+| D | <t:1667219160:D> | October 31, 2022 | Long Date |
+| f | <t:1667219160:f> | October 31, 2022 12:26 AM  | Short Date/Time |
+| F | <t:1667219160:F> | Monday, October 31, 2022 12:26 AM | Long Date/Time |
+| R | <t:1667219160:R> | 27 minutes ago | Relative Time |
+
+> 📌 The default style is `f`, if no style provided.
+
+
```
Comment (priyanujgogoi-28) [inline src/resources/discordTimestamps.md]:
```suggestion
```

File: src/resources/discordTimestamps.md [hunk 17]
```diff
@@ -0,0 +1,35 @@
...
+## Usability
+
```
Comment (priyanujgogoi-28) [inline src/resources/discordTimestamps.md]:
```suggestion
```

**Review by priyanujgogoi-28 — CHANGES_REQUESTED** [review]

**Review by priyanujgogoi-28 — APPROVED** [review]
Thanks!

**Comment by DevMarlin** [comment]
@MineBartekSA could I get a review on this?

**Review by MineBartekSA — APPROVED** [review]
Thank you for your hard work

**Pull request merged**